includes/options/class-amp-analytics-options-submenu.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/options/views/class-amp-analytics-options-submenu-page.php' );
+require_once( AMP__DIR__ . '/includes/utils/class-amp-html-utils.php' );
+
+class AMP_Analytics_Options_Submenu {
+
+	private $parent_menu_slug;
+	private $menu_slug;
+	private $menu_page;
+
+	public function __construct( $parent_menu_slug ) {
+		$this->parent_menu_slug = $parent_menu_slug;
+		$this->menu_slug = 'amp-analytics-options';
+		$this->menu_page = new AMP_Analytics_Options_Submenu_Page();
+	}
+
+	public function init() {
+		$this->add_submenu();
+		add_action(
+			'admin_head',
```

Comment:
Sorry I did not address this in the previous review; what is the standard way to achieve this? Could you point me to an example?